<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockOverview</class>
 <widget class="QDialog" name="StockOverview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_plaetze">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>391</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_produkt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>441</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>871</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_tableName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Table: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_platz">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>305</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Platz:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_produkt">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Produkt:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Platz">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Produkt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>560</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTables">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>580</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
